cycles.json

```diff
@@ -198,5 +198,12 @@
         "packs": [
             "AHaH"
         ]
+    },
+    {
+        "id": "spoils",
+        "name": "The Spoils of War",
+        "packs": [
+            "TSoW"
+        ]
```

Comment:
Is TSoW a standalone expansion, or will it be part of a cycle with more packs as part of the cycle? And if a cycle, should it be titled "The Spoils of War"?

(Only really matters for ThronesDB purposes, or if design team ever does a rotation of cycles)